build/generate_phars_list.php

```diff
@@ -0,0 +1,80 @@
+
+<?php
```

Comment:
```suggestion
#!/usr/bin/env php
<?php
```

Might also be a good idea to have a minimal file docblock, which at the very least identifies the file as part of PHP_CodeSniffer + annotates the minimum PHP version for this script (as it doesn't have to comply with the same PHP requirements the main package has).

Maybe something along the lines of:
```php
/**
 * Script to auto-generate the `phar/index.html` page for the GH Pages website.
 *
 * {@internal This script has a minimum PHP requirement of PHP ...}
 *
 * @copyright 2025 PHPCSStandards and contributors
 * @license   https://github.com/PHPCSStandards/PHP_CodeSniffer/blob/master/licence.txt BSD Licence
 */
```

Reply:
Done. I think the minimum PHP version should be PHP 7 because I used some type hints, but those could be removed if need be.